app.js

```diff
@@ -0,0 +1,94 @@
+    'United Arab Emirates', 'United Kingdom', 'United States', 'Uruguay', 'Uzbekistan', 'Vanuatu', 'Vatican City',
+    'Venezuela', 'Vietnam', 'Yemen', 'Zambia', 'Zimbabwe'];
+
+//Displays the total number of countiries in the header "h5"
+const totalCountry = document.querySelector("h5"); //target the h5 element
+totalCountry.textContent = `The toal number of countries are: ${myCountries.length}`; //give h5 content
+
+const parentDiv = document.getElementById('parentDiv'); //selected the parent element with class "row"
+// Function to display country
+function displayCountries(displayEachCountry) {//Take the countires as a parameter
+
+    parentDiv.innerHTML = "";//clear the parent div
+    //display new countries it receives
+    displayEachCountry.forEach((country) => {
+        const firstDiv = document.createElement('div'); // create the first div
+        firstDiv.classList = "col-sm-3"; //first div class of col-sm-2
+        const secondDiv = document.createElement('div'); //created the second div
+        secondDiv.className = "card"; //second div class of card
+        secondDiv.classList.add("mb-4") //second div with margin buttom 
+        const mainDiv = document.createElement('div'); //created the main div to house the content of the array
+        mainDiv.className = "card-body"; //class of card-body
+        mainDiv.classList.add("p-5") //class of padding
+        mainDiv.classList.add("text-center") //class of text center
+        mainDiv.classList.add('text-light') //text color white
+        mainDiv.textContent = country; //main div content
+
+        parentDiv.appendChild(firstDiv); //append the first div to the parent div
+        firstDiv.appendChild(secondDiv); //append the second div to the first div
+        secondDiv.appendChild(mainDiv); //append the main div to the second div 
+    })
+}
+displayCountries(myCountries); //called the function to display country
+
+//The Form Element
+const firstButton = document.forms['search'].querySelector('#starting-word'); //i selected the first button
+const secondButton = document.forms['search'].querySelector('#any-word')// i selected the second button
+
+const numberOfCountries = document.getElementById("display"); //i selected the p element with id display
+
+firstButton.addEventListener('click', function (e) { //add an event to the first button
+    e.preventDefault(); //prevents the page from refreshing
+
+    const value = document.forms['search'].querySelector('.input').value.toLowerCase(); //get the value of input
+    const listOfCountires = []; //initialise an empty array
+
+    myCountries.filter((country) => { //filter the array of countries
+        if (country.toLowerCase().startsWith(value)) { //if the country starts with the value inputed
+            listOfCountires.push(country); //push the countries to the listOfCountry array
+            displayCountries(listOfCountires); //hence display country with an argument of the listOfCountries array
+        }
+    })
```

Comment:
```suggestion
    myCountries.filter((country) => { //filter the array of countries
        if (country.toLowerCase().startsWith(value)) { //if the country starts with the value inputed
            listOfCountires.push(country); //push the countries to the listOfCountry array
        }
    })
    displayCountries(listOfCountires); //hence display country with an argument of the listOfCountries array
```
displayCountries should be called after going through all the array